src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -6791,12 +6791,6 @@ export class ZoneServer2016 extends EventEmitter {
       );
       return;
     }
-    const vehicle = this._vehicles[vehicleGuid];
-    if (vehicle._resources[ResourceIds.FUEL] + fuelValue > 10000) {
-      // Prevent players from overfilling their car to reserve fuel.
-      this.sendAlert(client, "Fuel level is not low enough to refuel");
-      return;
```

Comment:
Yep but by removing that players can now waste fuel trying to refuel while being at 100% the operation should fail in this case

Reply:
Ahh good point, Idk why I removed that check looking back 😂😂 Should be good to go